--- a/Makefile.am
+++ b/Makefile.am
@@ -8,6 +8,8 @@
 libtoaster/src \
 netconf/modules \
 example-modules/helloworld \
+example-modules/ietf-interfaces \
+example-modules/ietf-system \
 netconf/src/yangrpc \
 netconf/src/yangrpc/example \
 netconf/man \
--- a/configure.ac
+++ b/configure.ac
@@ -88,6 +88,8 @@
         netconf/src/subsys/Makefile
         libtoaster/src/Makefile
         example-modules/helloworld/Makefile
+        example-modules/ietf-interfaces/Makefile
+        example-modules/ietf-system/Makefile
         netconf/src/yangrpc/Makefile
         netconf/src/yangrpc/example/Makefile
         netconf/src/yangdump/Makefile
--- a/example-modules/ietf-system/Makefile.am
+++ b/example-modules/ietf-system/Makefile.am
@@ -4,11 +4,11 @@
 ietf-system.c
 
 #Integrated yuma123 build
-#libietf_system_la_CPPFLAGS = -I $(top_srcdir)/netconf/src/ -I$(top_srcdir)/netconf/src/agt -I$(top_srcdir)/netconf/src/mgr -I$(top_srcdir)/netconf/src/ncx -I$(top_srcdir)/netconf/src/platform -I$(top_srcdir)/netconf/src/ydump $(XML_CPPFLAGS)
-#libietf_system_la_LDFLAGS = -module $(top_builddir)/netconf/src/agt/libyumaagt.la $(top_builddir)/netconf/src/ncx/libyumancx.la
+libietf_system_la_CPPFLAGS = -I $(top_srcdir)/netconf/src/ -I$(top_srcdir)/netconf/src/agt -I$(top_srcdir)/netconf/src/mgr -I$(top_srcdir)/netconf/src/ncx -I$(top_srcdir)/netconf/src/platform -I$(top_srcdir)/netconf/src/ydump $(XML_CPPFLAGS)
+libietf_system_la_LDFLAGS = -module $(top_builddir)/netconf/src/agt/libyumaagt.la $(top_builddir)/netconf/src/ncx/libyumancx.la
 
 #Standalone mode. Depends on installed libyuma-dev
-libietf_system_la_CPPFLAGS = -I${includedir}/yuma/agt -I${includedir}/yuma/mgr -I${includedir}/yuma/ncx -I${includedir}/yuma/platform $(XML_CPPFLAGS)
-libietf_system_la_LDFLAGS = -module -lyumaagt -lyumancx
+#libietf_system_la_CPPFLAGS = -I${includedir}/yuma/agt -I${includedir}/yuma/mgr -I${includedir}/yuma/ncx -I${includedir}/yuma/platform $(XML_CPPFLAGS)
+#libietf_system_la_LDFLAGS = -module -lyumaagt -lyumancx
 
 #yang_DATA = ietf-system.yang
--- a/example-modules/ietf-interfaces/Makefile.am
+++ b/example-modules/ietf-interfaces/Makefile.am
@@ -4,12 +4,12 @@
 ietf-interfaces.c
 
 #Integrated yuma123 build
-#libietf_interfaces_la_CPPFLAGS = -I $(top_srcdir)/netconf/src/ -I$(top_srcdir)/netconf/src/agt -I$(top_srcdir)/netconf/src/mgr -I$(top_srcdir)/netconf/src/ncx -I$(top_srcdir)/netconf/src/platform -I$(top_srcdir)/netconf/src/ydump $(XML_CPPFLAGS)
-#libietf_interfaces_la_LDFLAGS = -module $(top_builddir)/netconf/src/agt/libyumaagt.la $(top_builddir)/netconf/src/ncx/libyumancx.la
+libietf_interfaces_la_CPPFLAGS = -I $(top_srcdir)/netconf/src/ -I$(top_srcdir)/netconf/src/agt -I$(top_srcdir)/netconf/src/mgr -I$(top_srcdir)/netconf/src/ncx -I$(top_srcdir)/netconf/src/platform -I$(top_srcdir)/netconf/src/ydump $(XML_CPPFLAGS)
+libietf_interfaces_la_LDFLAGS = -module $(top_builddir)/netconf/src/agt/libyumaagt.la $(top_builddir)/netconf/src/ncx/libyumancx.la
 
 #Standalone mode. Depends on installed libyuma-dev
-libietf_interfaces_la_CPPFLAGS = -I${includedir}/yuma/agt -I${includedir}/yuma/mgr -I${includedir}/yuma/ncx -I${includedir}/yuma/platform $(XML_CPPFLAGS)
-libietf_interfaces_la_LDFLAGS = -module -lyumaagt -lyumancx
+#libietf_interfaces_la_CPPFLAGS = -I${includedir}/yuma/agt -I${includedir}/yuma/mgr -I${includedir}/yuma/ncx -I${includedir}/yuma/platform $(XML_CPPFLAGS)
+#libietf_interfaces_la_LDFLAGS = -module -lyumaagt -lyumancx
 
 yang_DATA = \
 interfaces-notifications.yang
